weasyl/controllers/user.py

```diff
@@ -109,9 +111,9 @@ def signup_post_(request):
     form = request.web_input(
         username="", password="", passcheck="", email="", emailcheck="",
         day="", month="", year="", recaptcha_challenge_field="",
-        g_recaptcha_response=request.web_input("g-recaptcha-response"))
+        g_recaptcha_response="")
 
-    if not define.captcha_verify(form):
+    if not define.captcha_verify(form.g_captcha_response):
```

Comment:
``g_recaptcha_response``, or if you can get the dashes to work, ``g-recaptcha-response``. There's no form field named ``g_captcha_response``, either on the form itself or inserted by the JavaScript reCAPTCHA code.